content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -450,4 +450,70 @@ This annotation has been deprecated since Kubernetes v1.19 and will become non-f
 The tutorial [Restrict a Container's Syscalls with seccomp](/docs/tutorials/clusters/seccomp/) takes
 you through the steps you follow to apply a seccomp profile to a Pod or to one of
 its containers. That tutorial covers the supported mechanism for configuring seccomp in Kubernetes,
-based on setting `securityContext` within the Pod's `.spec`.
\ No newline at end of file
+based on setting `securityContext` within the Pod's `.spec`.
+
+<!-- kubeadm annotations, labels, taints -->
```

Comment:
Maybe we should instead put all these kubeadm annotations into a separate section?
From the annotation name (e.g. `node-role.kubernetes.io/control-plane`), we cannot tell whether such an annotation is created/used by kubeadm.

Reply:
I'm happy to see tagging (by what sets the label, etc) arrive later. I value having the annotations registered and documented, ahead of telling which in-project components commonly write or query them.

Reply:
@sftim Well ... I disagree. Sometimes when we say save something to future, it means that will never happen. It is nice to have a comment in the markdown source code, but that is not enough for readers.

Reply:
How about we log and freeze an issue to track the remaining bits of documentation work that need to happen?

Reply:
I can add a kubeadm section to this page. Perhaps even better might have been to have these additions in the kubeadm folders / content locations.

Reply:
The heading levels might need to change (top-level ## and subheadings ###) but other than that core (or Kubernetes) and kubeadm LGTM @neolit123